### PR TITLE
[TTS] Fix adapter duration issue

### PR DESCRIPTION
**Pull request by hsiehjackson**
# What does this PR do ?

Fix TTS adapter duration issue.

**Collection**: [TTS]

# Changelog 
- `examples/tts/conf/fastpitch_align_44100_adapter.yaml`
- `examples/tts/fastpitch_finetune_adapters.py`
- `nemo/collections/tts/losses/aligner_loss.py`
- `nemo/collections/tts/models/fastpitch.py`
- `tutorials/tts/FastPitch_Adapter_Finetuning.ipynb`

# Usage
* You can potentially add a usage example below

# Before your PR is "Ready for review"
**Pre checks**:
- [x] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [x] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

**Comment by subhankar-ghosh** [comment]
@hsiehjackson  Can you also add the code to increase loss scale for aligner loss may be. We can add these two methods in the tutorial later. I feel the "increasing loss scale for aligner loss" should also be a good approach especially because that doesn't add parameters to finetune.

**Comment by hsiehjackson** [comment]
> 



> @hsiehjackson Can you also add the code to increase loss scale for aligner loss may be. We can add these two methods in the tutorial later. I feel the "increasing loss scale for aligner loss" should also be a good approach especially because that doesn't add parameters to finetune.

@subhankar-ghosh Added

File: nemo/collections/tts/models/fastpitch.py [hunk 1]
```diff
@@ -138,9 +138,11 @@
 
         self.aligner = None
         if self.learn_alignment:
+            if "aligner_loss_scale" in cfg:
+                aligner_loss_scale = cfg.aligner_loss_scale
             self.aligner = instantiate(self._cfg.alignment_module)
-            self.forward_sum_loss_fn = ForwardSumLoss()
-            self.bin_loss_fn = BinLoss()
+            self.forward_sum_loss_fn = ForwardSumLoss(loss_scale=aligner_loss_scale)
```
[the remark this answers is not in the document]
Comment (subhankar-ghosh) [reply nemo/collections/tts/models/fastpitch.py:144]: Can you define `aligner_loss_scale=1.0` before assigning it. I think it gets initialized only inside the if condition.

**Comment by subhankar-ghosh** [comment]
1 Minor issue, remaining LGTM.

**Review by subhankar-ghosh — APPROVED** [review]
LGTM

**Pull request merged**